dist/amd/froala-editor.js

```diff
@@ -101,7 +101,7 @@ define(['exports', 'aurelia-framework', 'aurelia-binding', './froala-editor-conf
 				}
 			})];
 
-			this.editor = new _froala_editorPkgdMin2.default('#' + this.element.id + ' ' + editorSelector, Object.assign({}, this.config), function () {
+			this.editor = new _froala_editorPkgdMin2.default(this.element.querySelector(editorSelector), Object.assign({}, this.config), function () {
```

Comment:
it will select first element of that type. not that particular element?

Reply:
That's correct. However, since we are searching only within the element itself, and there is only one 'textarea', we will get that. 

This is similar to the old code:
        if (this.config.iframe) {
            this.instance = this.element.getElementsByTagName('textarea')[0];
        }
        else {
            this.instance = this.element.getElementsByTagName('div')[0];
        }